File: sdk/client/python/openapi-test/base/klay.py
from opensdk.sdk import OpenSDK
from base.constants import KLAYTN_URL


def create_new_filter():

    filterOptions = {
        "fromBlock": "earliest",
        "toBlock": "latest",
        "address": "0x87ac99835e67168d4f9a40580f8f5c33550ba88b",
        "topics": ["0xd596fdad182d29130ce218f4c1590c4b5ede105bee36690727baa6592bd2bfc8"]
    }

    sdk = OpenSDK(KLAYTN_URL)
    klay_response = sdk.klay.new_filter(filterOptions)

    return klay_response

File: sdk/client/javascript/openapi-test/helpers/klay.js
const OpenSdk = require("opensdk-javascript");
const { BAOBAB_RPC } = require("../test/constant");

const sdk = new OpenSdk(new OpenSdk.ApiClient(BAOBAB_RPC));

export const getKlayFilterId = () => {
    return new Promise((res, ej) => {
        const opts = {
            "fromBlock": "earliest",
            "toBlock": "latest",
            "address": "0x87ac99835e67168d4f9a40580f8f5c33550ba88b",
            "topics": [
                "0xd596fdad182d29130ce218f4c1590c4b5ede105bee36690727baa6592bd2bfc8"
            ]
        }
        sdk.klay.newFilter(opts, {}, (error, data, response) => {
            if (error) ej(error)
            return res(data)
        });
    })
}